package.json

```diff
@@ -17,6 +17,7 @@
     "blockfrost-local-test": "source ./test/blockfrost-local.env && spago run --main Test.Ctl.Blockfrost.Contract",
     "unit-test": "spago run --main Test.Ctl.Unit",
     "testnet-test": "spago run --main Test.Ctl.Testnet",
+    "bet-ref-test": "spago run --main Test.Ctl.BetRef",
```

Comment:
Let's make this test suite part of the testnet-test. I think it's nice to have a more involved example assuming it is stable and takes reasonable time to execute.